### PR TITLE
[docs] #249: Add the support channels

### LINKED ISSUE #249: Add the support channels to the documentation
Please correct me if I'm wrong, but it seems like we don't have contacts like Discord, Telegram or even issues mentioned in the docs. I think it's something deserving a small update.

### PR DESCRIPTION
**Pull request by 6r1d**
As mentioned in #249, I'd like to add the support channels in the documentation, so the users can contact us more quickly.

Closes #249 

Signed-off-by: 6r1d <vic.6r1d@gmail.com>

### CODE REVIEW

File: src/guide/support.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Receive support
+
+From time to time, you may have questions about Iroha that you would like to discuss in detail with others.
+There are three ways to quickly get in touch with our community: Telegram, Discord and GitHub.
```
Comment (outoftardis) [inline src/guide/support.md:4]:
```suggestion
There are three ways to quickly get in touch with our community: Telegram, Discord, and GitHub.
```

**Review by outoftardis — APPROVED** [review]

File: src/guide/support.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Receive support
+
+From time to time, you may have questions about Iroha that you would like to discuss in detail with others.
+There are three ways to quickly get in touch with our community: Telegram, Discord, and GitHub.
+
+A large part of the community currently uses [Telegram](https://t.me/hyperledgeriroha) for communication.
+The Hyperledger part of the team prefers [Discord](https://discord.gg/hyperledger), with two dedicated channels: `iroha` and `iroha-2-contributors`.
+The Discord and Telegram channels are synchronised, and users of both channels see your messages.
```
Comment (WRRicht3r) [inline src/guide/support.md:8]:
```suggestion
The Discord and Telegram channels are synchronized, so users of both media see your messages.
```

File: src/guide/support.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+Finally, you can [create](https://github.com/hyperledger/iroha/issues/new/choose) a GitHub issue, whether it's a request to update documentation, a suggestion for the core team, or a bug you have found.
```
Comment (WRRicht3r) [inline src/guide/support.md:10]:
```suggestion
Finally, you can create a [GitHub issue] (https://github.com/hyperledger/iroha/issues/new/choose), whether it's a request to update documentation, a suggestion for the core team, or a bug you have found.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/guide/support.md (hunk 3 above)
Comment (outoftardis) [reply src/guide/support.md:10]: or maybe?

```suggestion
Finally, you can [create a GitHub issue] (https://github.com/hyperledger/iroha/issues/new/choose), whether it's a request to update documentation, a suggestion for the core team, or a bug you have found.
```

Comment (WRRicht3r) [reply src/guide/support.md:10]: That looks much better

**Review by WRRicht3r — APPROVED** [review]

File: .vitepress/config.ts [hunk 4]
```diff
@@ -35,6 +35,10 @@ function getGuideSidebar(): DefaultTheme.SidebarGroup[] {
           text: 'Build and Install',
           link: '/guide/build-and-install',
         },
+        {
+            text: 'Receive support',
+            link: '/guide/support.md',
```
Comment (outoftardis) [inline .vitepress/config.ts:40]:
```suggestion
          text: 'Receive support',
          link: '/guide/support.md',
```

**Pull request merged**